ChurchCRM/CRM
# Feature/4015 add new calendar

#### What's this PR do?

Creates a "New Calendar" functionality to allow users to create new calendars

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/11679900/36460957-015ff638-1689-11e8-9f7f-326ea49c8984.png)

#### What Issues does it Close?

Closes #4015

#### Any background context you want to provide?
Part of the new calendar project

#### Where should the reviewer start?
Start with a new system installed from this branch's SQL installer.  Create a new calendar.

#### How should this be manually tested?
1) Create a new calendar

#### How should the automated tests treat this?
See the included test: https://github.com/ChurchCRM/CRM/pull/4016/files#diff-2264fd418dcd64dda0020bb5f284d202

#### Questions:
- Is there a related website / article to substantiate / explain this change?

- Does the development wiki need an update?

- Does the user documentation wiki need an update?

- Does this add new dependencies?

- Does this need to add new data to the demo database
